cmd/feed-dns/main.go

```diff
@@ -104,4 +116,9 @@ func validateConfig() {
 		log.Error("Must supply r53-hosted-zone")
 		os.Exit(-1)
 	}
+
+	if (internalIngressFQDN != "") != (externalIngressFQDN != "") { //XOR
```

Comment:
While it may be correct, this is very confusing syntax and could lead to maintainability issues in future - you may disagree, but I would recommend making this more explicit:

`if (internalIngressFQDN == "" && externalIngressFQDN != "") || (internalIngressFQDN != "" && externalIngressFQDN == "") {`